README.md

```diff
@@ -54,6 +54,39 @@ You can deploy the application to Kubernetes using the Linkerd 2.0 service mesh.
 
 ![Linkerd Dashboard](images/dashboard.png)
 
+## Running with MySQL ##
+
+The default booksapp configuration uses SQLite. It's also possible to run the
+app with a MySQL backend, using the configs in the `k8s/` directory.
```

Comment:
Maybe add: `Because this configuration runs mysql in a separate pod, it's possible to run multiple replicas of the app services`.